### PR TITLE
CSPL-2112 Initial changes for ES app

### PR DESCRIPTION
**Pull request by tpereirasplunk**
Edited ES test cases. 

- Added new scope 
- Updated with latest verification functions

### CODE REVIEW

File: test/appframework_aws/c3/appframework_aws_test.go [hunk 1]
```diff
@@ -911,7 +911,7 @@ var _ = Describe("c3appfw test", func() {
 			// Create App framework Spec
 			appSourceName := "appframework-shc-" + testenv.RandomDNSName(3)
 			appSourceVolumeNameShc := "appframework-test-volume-shc-" + testenv.RandomDNSName(3)
-			appFrameworkSpecShc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameShc, enterpriseApi.ScopeClusterWithPreConfig, appSourceName, s3TestDirShc, 60)
+			appFrameworkSpecShc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameShc, enterpriseApi.ScopePremiumApps, appSourceName, s3TestDirShc, 60)
```
Comment (akondur) [inline test/appframework_aws/c3/appframework_aws_test.go:914]: The appframework spec for the ES app has other parameters such as `premiumAppsProps` as well as `esDefaults` etc.. Are we planning to add this later to test various values?

**Review by akondur — APPROVED** [review]
In general, the handling of the ES app is different for standalone, SHC and IDXC. We should make sure to handle those scenarios during verification in the future.

File: test/appframework_aws/c3/appframework_aws_test.go (hunk 1 above)
Comment (gaurav-splunk) [reply test/appframework_aws/c3/appframework_aws_test.go:914]: +1

Comment (tpereirasplunk) [reply test/appframework_aws/c3/appframework_aws_test.go:914]: Added to the spec later

**Review by gaurav-splunk — APPROVED** [review]

**Review by jambrosiano — APPROVED** [review]

**Pull request merged**